backend/app/views/questions/show.slim

```diff
@@ -0,0 +1,32 @@
+h1
+  = @question.name
+h5
+  | This is a 
+  = @question.question_type.name.downcase
+  |  question.
+ul
+  - @question.options.each do |option|
+    - if @question.single_choice?
+      |  <input type='radio' id="
+      = option.name
+      | " name='
+      = @question.name
+      | ' value="
+      = option.name
+      | "> <label for="
+      =option.name
+      | "> 
+      = option.name
+    - elsif @question.multiple_choice?
+      |  <input type="checkbox" id="
+      = option.name
+      | " value="
+      = option.name
+      | "> <label for="
+      =option.name
+      | "> 
+      = option.name
+    - else
+      input[type="text"]
+button
+  | Next
```

Comment:
empty line, maybe there is a rule for this on rubocop or in some other place

Reply:
I think rubocop doesn't check front end files, maybe we could look for another gem that can help with that?

Reply:
https://github.com/espoo-dev/espoo-dev/pull/403/commits/324b55f0ba079997f70fc2e91067637d18887620